docs/docs/guides/context.md

```diff
@@ -0,0 +1,131 @@
+# Enrich Apps Traces with Context and Metadata
+
+This guide will walk you through the usage of Context in Keptn.
+
+After reading this guide you will be able to:
+
+- Add metadata to application and workloads traces
+- Link traces between different versions of your application
+
+`KeptnAppContext` is a custom resource in Keptn that allows you to add metadata
+and links to traces for a specific application. 
+This enables you to enrich your traces with additional information, 
+making it easier to understand and analyze
+the performance of your applications.
+
+Common usage of this feature may include:
+
+- adding commit id or references related to git-hops tooling
+- referring to different stages and actors
+
+To exploit this feature you need to install Keptn and deploy an application `app.yaml`,
+for instance like done in [step #1-#3 here](../getting-started/observability.md#step-1-install-keptn)
+
+To visualize traces you will require either Jaeger or Grafana:
+
+- Install
+  [Grafana](https://grafana.com/grafana/)
+  or the visualization tool of your choice, following the instructions in
+  [Grafana Setup](https://grafana.com/docs/grafana/latest/setup-grafana/).
```

Comment:
it reads weirdly, maybe we should shift around. Similarly for traces right after this bullet point
```suggestion
  following the instructions in [Grafana Setup](https://grafana.com/docs/grafana/latest/setup-grafana/)
  or the visualization tool of your choice.
```